scripts/busybox-ping-test/container_ops.py

```diff
@@ -53,7 +53,11 @@ def busybox_container_deploy(target_ips, ip_mac_db, container_names):
        command2 = "sudo ovs-docker add-port br-int eth1 " + con + " --ipaddress=" + db_ip + "/24" + " --macaddress=" + db_mac
        command3 = "sudo ovs-docker set-vlan br-int eth1 " + con + " 1"
        print("deploying busybox " + con + " on " + aca_ip)
-
+       if con == "con1":
+            gw = "10.0.1.1"
+       else:
+            gw = "10.0.2.1"
```

Comment:
Is it possible to make this configurable in the .ini file? I thought the gw ip was part of configurations already.

Reply:
Discussed with @pkommoju, the fix will be checked in in next PR for s5 fix.